changelog.d/5514.bugfix

```diff
@@ -0,0 +1 @@
+Make jinja2 and bleach required dependencies.
```

Comment:
Remember that the changelog is for the users' benefit, not ours. Please can you document change that the user will see, rather than the means by which we are achieving the change?

Reply:
Very good point - thanks for the reminder.